proxy/src/grpc/sql_query.rs

```diff
@@ -127,6 +136,9 @@ impl<Q: QueryExecutor + 'static> Proxy<Q> {
                     if tx.send(resp).await.is_err() {
                         error!("Failed to send affected rows resp in stream sql query");
                     }
+                    GRPC_HANDLER_COUNTER_VEC
+                        .query_succeeded_row
```

Comment:
How about defining a separate type of the metric for this? `query_affected_row`?